sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+# Generated by ./bqetl generate events_daily
+
+---
+friendly_name: 'Google Ad Campaign Cost Breakdown'
+description: >
+  A breakdown of google ad campaign costs per click, install, and DOU.
+owners:
+  - ctroy@mozilla.com
+labels:
+  application: fenix
+  schedule: daily
+  incremental: true
+scheduling:
+  dag_name: bqetl_campaign_cost_breakdowns
+  referenced_tables: [
+    [
+      'moz-fx-data-bq-fivetran',
```

Comment:
Is bigquery gonna know where this is?

Reply:
Sounds good. What steps do we/I need to take to make that happen?

Reply:
Already done! I moved it to be hourly